pkg/kubelet/preemption/preemption.go

```diff
@@ -189,10 +189,9 @@ func (a admissionRequirementList) distance(pod *v1.Pod) float64 {
 	dist := float64(0)
 	for _, req := range a {
 		remainingRequest := float64(req.quantity - resource.GetResourceRequest(pod, req.resourceName))
-		if remainingRequest < 0 {
-			remainingRequest = 0
+		if remainingRequest > 0 {
+			dist += math.Pow(remainingRequest/float64(req.quantity), 2)
```

Comment:
For a certain positive numbers why not remove math.Pow ?

` dist += remainingRequest/float64(req.quantity) `